test/InputSpec.js

```diff
@@ -240,7 +240,7 @@ describe('Input', function () {
       <Input type="text" label="l" wrapperClassName="wrapper" addonAfter="a" hasFeedback={true} help="h"/>
     );
 
-    let node = instance.getDOMNode();
+    let node = React.findDOMNode(instance);
```

Comment:
`React.findDOMNode(instance)` will grab the root DOM node for the react component regardless of what type of node it is. This is valuable for making assertions on the component's DOM tree. When we don't care about the layout of the DOM tree, then yes the other alternative will work.